### PR TITLE
Seperate general sections from architecture to partials

### PR DESCRIPTION
**Pull request by DebakelOrakel**

### CODE REVIEW

File: docs/modules/ROOT/pages/references/vsphere/architecture.adoc [hunk 1]
```diff
@@ -48,14 +49,7 @@ The bastion host must be provided by the vSphere infrastructure operator, but VS
...
-This network must be provided by the vSphere infrastructure operator.
-DHCP is mandatory for this network, but a number of IPs must be reserved to be used as Virtual IPs for the cluster.
-
-Traffic inside this network shouldn't be restricted.
-
-VMs in this network must be able to reach various services on the internet.
-See below for a detailed list of external systems that must be reachable.
+include::partial$architecture/networking-cluster.adoc[]
 
 === Virtual IPs
```
Comment (DebakelOrakel) [inline docs/modules/ROOT/pages/references/vsphere/architecture.adoc:54]: This probably could be parameterized 🤔

Comment (DebakelOrakel) [reply docs/modules/ROOT/pages/references/vsphere/architecture.adoc:54]: ...or not,  its too infra specific.

File: docs/modules/ROOT/pages/references/vsphere/architecture.adoc [hunk 2]
```diff
@@ -73,29 +59,11 @@ These additional VIPs will be managed by `keepalived` instances on the cluster.
...
-APPUiO Managed OpenShift 4 uses `172.30.0.0/16` as the service network.
-
-Both of these networks are interanl to the OpenShift 4 cluster.
-Therefore, the IP CIDRs for these networks must not be routable from the outside.
-Additionally, the same IP CIDRs can be reused for multiple OpenShift 4 clusters.
-
-However, the chosen CIDRs shouldn't overlap with existing networks allocated by the customer.
-If there are overlaps, external systems in the overlapping ranges won't be accessible from within the OpenShift 4 cluster.
-The pod and service network CIDRs can be customized if and only if there are conflicts.
+include::partial$architecture/networking-pod.adoc[]
```
Comment (haasad) [inline docs/modules/ROOT/pages/references/vsphere/architecture.adoc:62]:
```suggestion
include::partial$architecture/networking-pods.adoc[]
```

**Review by haasad — APPROVED** [review]
I checked that the vsphere docs still look (almost) the same and rendered correctly. LGTM. Everything else will probably need a few rounds of iterative improvements anyway. Thanks a lot for this work :handshake:

**Pull request merged**